embedding-store-filter-parsers/langchain4j-embedding-store-filter-parser-sql/src/main/java/dev/langchain4j/store/embedding/filter/builder/sql/ColumnDefinition.java

```diff
@@ -0,0 +1,39 @@
+package dev.langchain4j.store.embedding.filter.builder.sql;
+
+import dev.langchain4j.Experimental;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+import static dev.langchain4j.internal.ValidationUtils.ensureNotBlank;
+
+@Experimental
+@ToString
+@EqualsAndHashCode
+public class ColumnDefinition {
+
+    private final String name;
+    private final String type;
```

Comment:
Hmm. I did not think about using enum for this, but yes, since this class is used only to provide LLM with the table structure, we do not have to have strict types and user should have a full freedom here.

Reply:
Yeah, let's keep it that way, as it's not restricting possible types that could be used. That's fine.